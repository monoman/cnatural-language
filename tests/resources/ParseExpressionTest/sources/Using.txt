() => {
	using (var s = new FileStream(@"c:\file.txt")) {
		s.Read();
	}
}