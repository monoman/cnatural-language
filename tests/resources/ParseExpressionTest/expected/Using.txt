Lambda( => {
using (var s = new FileStream(Literal(VerbatimString, [@"c:\file.txt"]));
) {
(Name(s)).Read();
}
}
)